### ISSUE TITLE
Submission File wizard doesn't suggest filename to secondary locales

### ISSUE THREAD

**Issue by Vitaliy-1** [pkp/pkp-lib#6661]
**Describe the bug**
Multilingual OJS installation; submission file upload, including dependent files. The file name is suggested only for a primary locale.
1. During the file upload process, `Review Details` tab, The `Continue` button is active and it's possible to continue file upload even though even if file name is populated for only one locale.
![screencapture-localhost-8000-index-php-psp-workflow-index-262-1-2021-01-21-16_54_06](https://user-images.githubusercontent.com/20665587/105367717-69895100-5c09-11eb-8189-5b98a920fd2d.png)
 2.  During the file upload process, `Review Details` tab, click on the file name input field, multilingual inputs become available, click outside the input fields -> `Continue` button becomes inactive, it's impossible to start upload while the field(s) on a secondary locale isn't populated. 
![screencapture-localhost-8000-index-php-psp-workflow-index-262-1-2021-01-21-16_58_12](https://user-images.githubusercontent.com/20665587/105368161-e6b4c600-5c09-11eb-82c5-fdb2b0ed203d.png)

**What application are you using?**
 OJS 3.3

**Additional information**
I would expect consistency here, either allow uploading a submission file without populated file name input for all locales or disallow. As for autosuggests, I would prefer if they were implemented for secondary locales of the multilingual file name inputs.

**Comment by NateWr** [comment]
Yes, I discovered this as well but I don't think it is catching too many people out. Files should only require a filename in the primary (submission) locale. (In the long-term, I'd like filename to not be a multilingual field, but this will required changing publication formats in OMP because they use the filename directly.)

**Comment by asmecher** [comment]
@Vitaliy-1, can you verify whether https://github.com/pkp/pkp-lib/issues/7369 fixed this issue?
